python/deltalake/_internal.pyi

```diff
@@ -171,13 +171,15 @@ def write_new_deltalake(
 def write_to_deltalake(
     table_uri: str,
     data: pyarrow.RecordBatchReader,
+    data_schema: pyarrow.Schema,
```

Comment:
Why does this argument need to be added? the `RecordBatchReader` should already have a schema associated with it which can be used

Reply:
Hm, true. I'll change that.